src/main/java/com/smartling/marketo/sdk/rest/transport/JaxRsHttpCommandExecutor.java

```diff
@@ -51,6 +52,10 @@ public class JaxRsHttpCommandExecutor implements HttpCommandExecutor {
             "607",  // Daily quota
             "615"   // Concurrent request limit
     );
+    private static final Set<String> UPDATE_CONTENT_NOT_ALLOWED_CODES = ImmutableSet.of("702");
+    private static final String UPDATE_CONTENT_NOT_ALLOWED_MESSAGE_START = "Update content";
+    private static final String UPDATE_CONTENT_NOT_ALLOWED_MESSAGE_END = "not allowed.";
+
     private static final Set<String> NOT_FOUND_CODES = ImmutableSet.of("702", "710");
```

Comment:
looks like, NOT_FOUND 702 code is now treated as NOT_ALLOWED, maybe remove it from here?

Reply:
I'm not sure, that 702 code should be removed from NOT_FOUND. In the Marketo documentation, it describes as ```702	No data found for given search scenario	No records matched the given search parameters.```

Reply:
my mistake